tests/app/main/test_briefs.py

```diff
@@ -532,6 +532,23 @@ def test_essential_requirements_met_question_replays_all_brief_requirements(self
         for index, requirement in enumerate(self.brief['briefs']['essentialRequirements']):
             assert requirement == list_items[index]
 
+    def test_essential_requirements_met_question_escapes_brief_data(self):
+        self.brief['briefs']['essentialRequirements'] = [
+            '<h1>Essential one with xss</h1>',
+            '**Essential two with markdown**'
+        ]
+        self.data_api_client.get_brief.return_value = self.brief
+
+        res = self.client.get(
+            '/suppliers/opportunities/1234/responses/5/do-you-have-all-the-essential-skills-and-experience'
+        )
+        assert res.status_code == 200
+
+        doc = html.fromstring(res.get_data(as_text=True))
+        list_items = doc.xpath("//*[@id='input-essentialRequirementsMet-question-advice']/ul/li/text()")
+        assert list_items[0] == '<h1>Essential one with xss</h1>'
```

Comment:
Not sure if it would make the test clearer to assert the escaped version is in the page - i.e. `&lt;h1&gt;Essential one with xss&lt;/h1&gt;`.
Although it's nicer to render the HTML and have the xpath look in the right place, so maybe how it is is fine.